apps/web/components/templates/challenges/all/allChallenges.tsx

```diff
@@ -0,0 +1,44 @@
+import { ChallengeItem } from 'molecules/challengeItem/challengeItem';
+
+type challenge = {
+	title: string;
+	description: string;
+	designs: string;
+	difficulty: string;
+	image: string;
+};
```

Comment:
In TypeScript it's common to have type definitions that starts with uppercase (Strictly speaking in [PascalCase](https://pl.wikipedia.org/wiki/PascalCase)). So in this example it could be called `Challenge`
As I see, this pattern is being used in this project too. So it might be worth to stick to conventions 😅

Reply:
You are right, i don't saw this and i forgot about this because people said to me about this earlier.